removeAllureFolder.ts

```diff
@@ -0,0 +1,11 @@
+import {promises as fs} from 'fs';
+
+const deleteAllureResults = async () => {
+  try {
+    await fs.rm('allure-results', {recursive: true});
+  } catch (error) {
+    /* empty */
+  }
+};
+
+deleteAllureResults();
```

Comment:
`deleteAllureResults` is an asynchronous function that should be called with `await` keyword inside another asynchronous method. Besides, if you call the function directly after you declared it, it will be called first (as a Higher Order Function) even before your `pretest` script is executed. Then, your script tries to execute this function again.

Reply:
Im running `npx ts-node removeAllureFolder.ts` command to delete `allure-results` folder 
in terminal 
how can I wrap this in another async function or should I call this function inside immediately invoked function expression

Reply:
If you call it inside IIFE, then may not need the script. Because, it should delete the folder and there's no use of trying to delete already deleted folder.